ui/src/components/ResourceConfigForm/ParameterInput/IntParamInput.tsx

```diff
@@ -45,6 +45,15 @@ const IntParamInputComponent: React.FC<ParamInputProps<number>> = ({
               {errors[definition.name]}
             </FormHelperText>
           )}
+          {definition.documentation && (
+            <Stack component={"span"}>
+              {definition.documentation.map((d) => (
+                <a href={d.url} rel="noreferrer" target="_blank">
```

Comment:
```suggestion
                <a href={d.url} rel="noreferrer" target="_blank" key={d.url}>
```
If each item doesn't have a key we get a console error (https://github.com/observIQ/bindplane-op/pull/1175)